templates/4_Post_Clustering.ipynb

```diff
@@ -6,9 +6,10 @@
    "metadata": {},
```

Comment:
<p>Should be section 4.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>